src/bin/pgcopydb/dump_restore.c

```diff
@@ -188,6 +188,16 @@ copydb_target_prepare_schema(CopyDataSpec *specs)
 		return false;
 	}
 
+	/*
```

Comment:
I just found that the `copydb_prepare_extensions_restore` should be executed after the `pg_restore_db`. 

i.e. after the pre-data restore

All other changes looks good.

Reply:
oh

Reply:
done

Reply:
The problem is the pre-data restore is executing `ALTER TABLE _timescaledb_internal._hyper_1_1_chunk OWNER TO postgres` on the hypertable chunk which is not supported.

Adding `--no-owner` to clone would help.

Reply:
when the pre_restore() function is called before then this is not occurring...

Reply:
how do we make this compatible with owners though?

Reply:
>when the pre_restore() function is called before then this is not occurring...

If you want to reproduce the error, use `timescale/timescaledb:2.13.0-pg13` docker image from compose.yaml.

>how do we make this compatible with owners though?

I don't think that is possible as of now.